machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -53,18 +53,18 @@ static void Main(string[] args)
             Console.WriteLine("=============== End of process ===============");
         }
 
-        public static TrainCatalogBase.TrainTestData LoadData(MLContext mlContext)
+        public static TrainTestData LoadData(MLContext mlContext)
         {
 
             //Note that this case, loading your training data from a file, 
             //is the easiest way to get started, but ML.NET also allows you 
             //to load data from databases or in-memory collections.
             // <SnippetLoadData>
-            IDataView dataView = mlContext.Data.LoadFromTextFile<SentimentData>(_dataPath,hasHeader:false);
+            IDataView dataView = mlContext.Data.LoadFromTextFile<SentimentData>(_dataPath, hasHeader: false);
             // </SnippetLoadData>
 
             // <SnippetSplitData>
-            TrainCatalogBase.TrainTestData splitDataView = mlContext.BinaryClassification.TrainTestSplit(dataView, testFraction: 0.2);
+            TrainTestData splitDataView = mlContext.Data.TrainTestSplit(dataView, testFraction: 0.2);
```

Comment:
>testFraction: 0.2); [](start = 82, length = 19)

do you think it is necessary to comment about what this is doing?

Reply:
yes! good catch.